server-side-scanners/boostsecurityio/snyk-connector/module.yaml

```diff
@@ -0,0 +1,2 @@
+name: Snyk Connector
+namespace: boostsecurityio/snyk-connector
```

Comment:
In the exoscanner code, we called this scanner:  `boostsecurityio/snyk-provider`. Should it be `provider` or `connector`?.

Same for the name, it's just `Snyk` at the moment.

Reply:
we can call it snyk-provider, but the FE will need to change the ID
for the name we should distinguisk the snyk provider from scanner